policy/pa_test.go

```diff
@@ -405,7 +405,7 @@ func TestChallengesForWildcard(t *testing.T) {
 	test.AssertError(t, err, "ChallengesFor did not error for a wildcard ident "+
 		"when DNS-01 was disabled")
 	test.AssertEquals(t, err.Error(), "Challenges requested for wildcard "+
-		"identifier but DNS-01 challenge type is not enabled")
+		"identifier but DNS based challenge type is not enabled")
```

Comment:
You'll probably want to add `core.ChallengeTypeDNSAccount01: false,` to the `enabledChallenges` map on line 399. Playing with that, I enabled the new challenge and received 
```
The ACME server refuses to issue a certificate for this domain name, because it is forbidden by policy
Domain name has more than one wildcard
--- FAIL: TestChallengesForWildcard (0.00s)
    pa_test.go:407: ChallengesFor did not error for a wildcard ident when DNS-01 was disabled: expected error but received none
```